### ISSUE TITLE
"Prevent proceed" function does not work in Chinese, Japanese, Korean environments

### ISSUE THREAD

**Issue by samumist** [h5p/h5p-branching-scenario#133]
## Issue Description
The "Prevent proceed" function does not work in Chinese, Japanese, Korean environments. However, it works fine in English, German, and Spanish, of the few language I tested.

## Steps to Reproduce
1. Set the language of the environment to Chinese, Japanese, or Korean with LUMI.
2. Create a branching course with only 1 video.
3. Enable the "Require to watch" option for the video.
4. Attempt to use the "Prevent proceed" function.
5. Observe that the function does not work as expected.

## Expected Behavior
The "Prevent proceed" function should disable the "proceed" button. It doesn't in Chinese language environment.
<img width="982" alt="截屏2023-06-15 17 35 23" src="https://github.com/h5p/h5p-editor-branching-scenario/assets/3883086/4cf932f0-fe8a-4e8b-a5c4-a6889121e4b8">

## Actual Behavior
The "Prevent proceed" function does not disable the 'proceed' button.
Same course, opened in an English/German/Spanish Environment, the 'proceed' button is disabled.
<img width="996" alt="截屏2023-06-15 17 36 10" src="https://github.com/h5p/h5p-editor-branching-scenario/assets/3883086/e6cdf708-62eb-4a22-9d35-7205fd5c1373">

**Comment by otacke** [comment]
Just amending:

I can reproduce the behavior on LUMI if I set LUMI's language to Japanese for instance (the language of the content does not matter). There are no messages on the development console.

However:
- I cannot reproduce the behavior on WordPress if I set WordPress' language to Japanese for instance (the language of the content does not matter). 
- I cannot reproduce the behavior on moodle (using moodle's custom integration) if I set moodle's language to Japanese for instance (the language of the content does not matter).
- I have not tried moodle or Drupal using the official H5P plugin or something else.

This may be some strange, yet interesting issue in LUMI, not Branching Scenario - or maybe something that occurs when combining both that either side could fix/work around.

@samumist You should H5P related report bugs at https://h5p.org/forum/9 not here (here reports tend to get not get handled). You can report issues with LUMI using the "Report issue" item in LUMI's "HELP" menu.

**Comment by samumist** [comment]
Many thanks @otacke 
It's very likely a problem of Lumi. 
I encountered a similar problem with another content type of yours, the "Image Choice Round" which works fine on WordPress but not on Lumi. 

I'll run some more tests on different platforms to verify if it is indeed a Lumi problem and report the issue accordingly.


Thank you, @otacke, for your help. It appears that the issue may be related to Lumi. I have experienced a similar problem with another content type you created, specifically the 'Image Choice Round' which functions properly on WordPress but not on Lumi.

To confirm whether this is a Lumi-specific issue, I will conduct additional tests on various platforms. If my findings support this hypothesis, I will report the issue to LUMI.
